runtime/autoload/download.lua

```diff
@@ -0,0 +1,41 @@
+basepath = "http://ftp.vim.org/pub/vim/runtime/spell/"
+spell_home = "~/.config/nvim/spell/"
+
+--[
+-- Checks if a file or directory exists.
+-- ]
+function exists(path)
+  local file = os.execute("test -f "..path)
+  local dir = os.execute("test -d "..path)
+  return (dir or file) == true
+end
+
+--[
+-- Downloads a specific language file from the vim servers.
+-- @Asserts correct lang, spell_home, basepath and internett connection.
+--]
+function download(url, dest)
+  if exists(spell_home..lang) then
+    print("file exists")
+  else
+    if not io.popen("which curl &> /dev/null") then
```

Comment:
`&>` is a bash feature, wouldn't work in POSIX shell.

Reply:
Doesn't work in cmd.exe so it won't work on Windows.

Reply:
Do not use lua built-in library here at all, VimL functions like `executable()`, `system([…])` (note: with argument list in order to not bother with escaping), `mkdir()`, `isdirectory()` are better suited here. These are available via API (vim.api.nvim_call_function), but I would suggest to drop lua completely and use VimL instead, this is not a thing which is easier to write in lua.

Basically you must run exactly one command: curl. For everything else there is VimL function. And run it via `system()`.